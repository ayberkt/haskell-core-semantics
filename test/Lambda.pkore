app(lam(tmVar(<type>, name(foo)), var(tmVar(<type>, name(foo)))), lit(litInt(17, <type>)))
