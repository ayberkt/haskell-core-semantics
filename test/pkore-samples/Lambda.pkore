app(lam(tmVar([type omitted], name(foo)), var(tmVar([type omitted], name(foo)))), lit(litInt(17, [type omitted])))
