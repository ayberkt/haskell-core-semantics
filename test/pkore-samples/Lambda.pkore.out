<generatedTop>
  <k> val ( 2 ) </k>
  <store>
    1 |-> closure ( name ( foo ) , var ( tmVar ( [type omitted] , name ( foo ) ) ) , .Map )
    2 |-> litObj ( lit ( litInt ( 17 , [type omitted] ) ) )
  </store>
  <env>
    .Map
  </env>
  <exit-code> 0 </exit-code>
</generatedTop>
